src/constants/index.js

```diff
@@ -1,2 +1,14 @@
 export { HypothesisStatuses } from "./hypothesis-statuses";
 export { IncidentPriority } from "./incident-priority";
+
+export const NodePodPropToLabelMap = {
+  node: "Node",
+  created: "Created At",
+  "instance-type": "Instance Type",
+  zone: "Zone",
+  os: "OS",
+  containerRuntime: "Container Runtime",
+  kernel: "Kernel",
+  kubeProxy: "Kubernetes proxy",
+  kubelet: "Kubelet "
+};
```

Comment:
If this is only used at one place, might be better to keep in the file where it's used. Otherwise, while reading code, one would have to jump files to figure what the map is.
UTY.